### DISCUSSION TITLE
Support for Adobe Acrobat Packed PDFs?

### DISCUSSION THREAD

**Discussion by PaulWeustink** [sumatrapdfreader/sumatrapdf#4153] [Ideas]
Adobe has the option to pack multiple PDFs into one PDF.
When opening such a packed PDF, SumatraPDF already shows the bookmarks of the files it contains.
With the web browser (Firefox) I am able to see those (and save them locally).
Is it an idea to make the bookmarks work on packed PDF files as well?

**Comment by GitHubRulesOK** [comment]
It did work after a fashion in earlier version 3.1.2 but the format is an exotic form of nesting "attachments" hence they are handled like a nested Docx or XML or SWF (Now A DEAD unsecure format) and really for security all should be considered as separate entities.

Clearly internal PDF is just as safe as the outer shell, but a lot of work needed to maintain links when temporarily extracted, so for now is handled as similar to other nested vipers.

Still an unclosed open issue. just for now cant remember the issue number

**Reply by YDX-2147483647** [reply]
> Still an unclosed open issue. just for now cant remember the issue number

I guess it should be #1602.

**Comment by kjk** [comment]
Please open a new issue and include a sample PDF.
